staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1263,6 +1275,12 @@ func TestListContinuation(t *testing.T) {
 		t.Logf("continue token was %d %s %v", rv, key, err)
 		t.Fatalf("Unexpected second page: %#v", out.Items)
 	}
+	if transformer.reads != 3 {
```

Comment:
I might feel slightly better if you reset the transformer reads count before you listed

Reply:
Done.